src/Component/Projects/addProject.tsx

```diff
@@ -0,0 +1,216 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import Modal from "@mui/material/Modal";
+import Select from "@mui/material/Select";
+import MenuItem from "@mui/material/MenuItem";
+import { useFormik } from "formik";
+import { useDispatch } from "react-redux";
+import * as yup from "yup";
+import { addProject } from "../../redux/slice/project/addProjectSlice";
+const style = {
+  position: "absolute" as "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: "50%",
+  bgcolor: "background.paper",
+  border: "2px solid #000",
+  boxShadow: 24,
+  p: 4
+};
+const validationSchema = yup.object({
+  ProjectName: yup.string().required("name is required"),
+  Username: yup.string().required("Username is required"),
+  Description: yup.string().required("Description is required"),
+  Link: yup.string().required("Link is required"),
+  Rate: yup.string().required("Rate is required"),
+  Team: yup.string().required("team is required")
+});
+const AddProject: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  const teamDetails = [
```

Comment:
create seperate constants for every hardcoded values

Reply:
fixed